docs/global-components/qr-color-table.vue

```diff
@@ -26,8 +27,9 @@ const rows = computed(() => props.list ?? data[dataTitle.value])
         <td v-if="desc === null" colspan="2">Unsupported</td>
         <template v-else>
           <td>
-            <div :class="[cls, { 's-bg-primary': /^s-text-inverted/.test(cls) },{ 'border': !/^s-bg/.test(cls) }, { 'h-24': !/^s-text/.test(cls) } ]" class="w-64 px-8">
+            <div :class="[cls, { 's-bg-inverted': /^s-(text|icon)-(inverted)/.test(cls) }, { 's-bg-notification': /^s-text-notification/.test(cls) }, { 'border': !/^s-bg/.test(cls) }, { 'h-24': !/^s-(text|icon)/.test(cls) } ]" class="w-64 px-8 text-center">
```

Comment:
Small thing, but unnecessary parentheses? 🙂  
```suggestion
            <div :class="[cls, { 's-bg-inverted': /^s-(text|icon)-inverted/.test(cls) }, { 's-bg-notification': /^s-text-notification/.test(cls) }, { 'border': !/^s-bg/.test(cls) }, { 'h-24': !/^s-(text|icon)/.test(cls) } ]" class="w-64 px-8 text-center">
```